clusterman/tools/rookout.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from os import getenv
+
+
+def enable_rookout():
```

Comment:
```suggestion
def enable_rookout() -> None:
```

dunno how i missed that in the original review :p

Reply:
do'h me neither